sdk/textanalytics/ai-text-analytics/src/textAnalyticsClient.ts

```diff
@@ -383,8 +384,8 @@ export class TextAnalyticsClient {
        * earlier versions were throwing an exception that included the inner
        * code only.
        */
-      const innerCode = e.response?.parsedBody?.error?.innererror?.code;
-      const innerMessage = e.response?.parsedBody?.error?.innererror?.message;
+      const innerCode = e.response?.parsedBody?.error?.innerError?.code;
```

Comment:
This should still be `innererror`

Reply:
Thought we weren't exposing this in the API surface at all other than, in JS, through our RestError. Here it's used just for the handling of the "too many documents" error.

Reply:
maybe i'm confused about what this code is handling, but it appears to be getting the inner error of the returned error, and since the error returned by the service is still `innererror`, I'm confused why the need to change here and access `innerError` instead (in python at least this would cause an error)

Reply:
@iscai-msft this is incorrect link. I removed renaming transformations from the swagger file and renamed those back.